[*] Running ...
[*] Dataset loaded: /home/dokhanhnam1199/QD/problems/tsp_aco/dataset/train50_dataset.npy with 5 instances.
[*] Instance 0: 5.8859341456485685
[*] Instance 1: 6.251603098537423
[*] Instance 2: 7.131534911862602
/home/dokhanhnam1199/QD/problems/tsp_aco/gpt.py:40: RuntimeWarning: overflow encountered in exp
/home/dokhanhnam1199/QD/problems/tsp_aco/gpt.py:41: RuntimeWarning: invalid value encountered in divide
Traceback (most recent call last):
  File "/home/dokhanhnam1199/QD/problems/tsp_aco/eval.py", line 53, in <module>
    obj = solve(node_pos)
  File "/home/dokhanhnam1199/QD/problems/tsp_aco/eval.py", line 26, in solve
    heu = heuristics(dist_mat.copy()) + 1e-9
  File "/home/dokhanhnam1199/QD/problems/tsp_aco/gpt.py", line 42, in heuristics_v2
  File "numpy/random/mtrand.pyx", line 990, in numpy.random.mtrand.RandomState.choice
ValueError: probabilities contain NaN
11
1
131.76952268336282
